app/controllers/mixins/middleware_common_mixin.rb

```diff
@@ -9,10 +8,19 @@ def index
     redirect_to :action => 'show_list'
   end
 
+  def show
+    return unless init_show
+    show_middleware
+  end
+
   private
 
-  def display_name
-    ui_lookup(:tables => @record.class.base_class.name)
+  def display_name(display = nil)
+    if display.nil?
```

Comment:
perhaps more suitable to check here for `.blank?`  which covers also an empty string ?